frontend/server/src/Controllers/Course.php

```diff
@@ -1451,9 +1488,9 @@ public static function getIntroDetails(\OmegaUp\Request $r) : array {
                 $group
             );
             $isFirstTimeAccess =
-                $sharingInformation['share_user_information'] == null;
+                $sharingInformation['share_user_information'] === null;
```

Comment:
nit:

```suggestion
	                is_null($sharingInformation['share_user_information']);
```

igual en todos los demás lugares de este cambio donde comparas con `null`.

Reply:
Listo